### PR TITLE
feat: Redesign filters

### PR DESCRIPTION
**Pull request by pavloburchak**
This PR...
image

## Changes

- added status and label filters
- redesigned menu of filters according mock-up
- removed applied labels

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {StyledOkButton, StyledResetButton, StyledSpace} from './FilterMenuFooter.styled';
+
+type Props = {
```
Comment (Toxer10) [inline src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx:3]: Rename Props to FilterMenuFooterProps, please

File: src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  onReset: () => void;
+  onOk: () => void;
+};
+
+const FilterMenuFooter: React.FC<Props> = ({onReset, onOk}) => {
```
Comment (Toxer10) [inline src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx:8]: Use here `props` argument and destructure it below

File: src/components/molecules/FilterMenu/MenuFilter.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    .ant-checkbox-inner {
+      border-color: ${Colors.purple};
+    }
+  }
+`;
+
+export const StyledFilterDropdown = styled(Dropdown)`
+  padding: 0 4px;
+
+  ${({visible}) => visible && `background-color: ${Colors.greyHover};`}
```
Comment (Toxer10) [inline src/components/molecules/FilterMenu/MenuFilter.styled.tsx:42]: 1) Rename visible => isVisible
2) Use props => props.isVisible
3) Use ternary, not &&

File: src/components/molecules/StatusFilter/StatusFilter.tsx [hunk 4]
```diff
@@ -37,33 +45,39 @@ const StatusFilter: React.FC<FilterProps> = props => {
...
     return statusList.map(status => {
       return (
-        <Menu.Item key={status}>
-          <Checkbox checked={filters.status === status} onChange={() => handleClick(status)}>
-            {status}
-          </Checkbox>
-        </Menu.Item>
+        <StyledFilterMenuItem key={status}>
+          <StyledFilterCheckbox checked={filters.status === status} onChange={() => handleClick(status)}>
+            {status[0].toUpperCase().concat(status.slice(1))}
```
Comment (Toxer10) [inline src/components/molecules/StatusFilter/StatusFilter.tsx:53]: Move this to string utils

File: src/components/molecules/FilterMenu/MenuFilter.styled.tsx (hunk 3 above)
Comment (pavloburchak) [reply src/components/molecules/FilterMenu/MenuFilter.styled.tsx:42]: fixed

File: src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx (hunk 1 above)
Comment (pavloburchak) [reply src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx:3]: done

File: src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx (hunk 2 above)
Comment (pavloburchak) [reply src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx:8]: done

**Review by Toxer10 — APPROVED** [review]

File: src/components/molecules/StatusFilter/StatusFilter.tsx (hunk 4 above)
Comment (pavloburchak) [reply src/components/molecules/StatusFilter/StatusFilter.tsx:53]: done

**Pull request merged**